.jenkins/Jenkinsfile-wheel-builder

```diff
@@ -0,0 +1,150 @@
+def configs = [
+    [
+        label: 'windows',
+        versions: ['py26', 'py27', 'py34', 'py35', 'py36'],
```

Comment:
Can you go find what %age of windows wheels are 3.3 so I can pretend to be data driven when I say "awesome, let's drop it"?